reana_db/utils.py

```diff
@@ -22,7 +22,14 @@ def build_workspace_path(user_id, workflow_id=None):
     :return: String that represents the workspace relative path.
         i.e. users/0000/workflows/0034
     """
-    workspace_path = os.path.join("users", str(user_id), "workflows")
+    from reana_commons.config import SHARED_VOLUME_PATH
+
+    if shared_volume:
+        workspace_path = os.path.join(shared_volume, "users", str(user_id), "workflows")
```

Comment:
As we discussed, we don't want to append "/users/{user_id}/workflows/" to the user specified custom workspace path. So I guess in this line we could only set `workspace_path = shared_volume`. But then probably the `shared_volume` variable name is not entirely correct, maybe `workspace_root_path` could be used as in `reana-workflow-controller`?

Also this function misses param docstring in line ~20

Reply:
In the original case when I have `var/reana` should "/users/{user_id}/workflows/" also be deleted?